uk_bin_collection/uk_bin_collection/councils/BromleyBoroughCouncil.py

```
@@ -30,7 +30,8 @@ def parse_data(self, page: str, **kwargs) -> dict:
             data = {"bins": []}
 
             # Get our initial session running
-            driver = create_webdriver(web_driver, headless, None, __name__)
+            user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
+            driver = create_webdriver(web_driver, headless, user_agent, __name__)
```

Ruff (0.15.0)

[error] `create_webdriver` may be undefined, or defined from star imports

(F405)